pages/asks.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useMemo } from "react";
+import styles from "@/styles/pages/asks.module.scss";
+import Ask from "@/components/Ask";
+import content from "@/server/content";
+import CreateAskForm from "@/components/CreateAskForm";
+import Modal from "@/components/Modal";
+
+export default function Asks() {
+	const contentArr = Object.entries(content);
```

Comment:
Commented before, splitting the `Object.entries` from the calculation of the list's elements introduce more confusion than clarity as its purpose is separated from the implementation. Code units in general should be colocated to other related units.

* Also it bring an hindrance on performance that you tried to prevent. 
Be aware that this function runs every re-render, even when it shouldn't (clicking on 'ask' button).

Bottom line: call this inside the useMemo

Reply:
noted